docs/en/02_Creating_pages_and_content/01_Creating_And_Editing_Content/02_Previewing_Changes.md

```diff
@@ -29,3 +29,12 @@ You can either:
 This will allow you to view a preview of your draft/live content alongside the Page management section. That way you can make updates and see the preview immediately.
 
 ![Preview option](/_images/Preview.png)
+
+
+## Viewing the site in draft mode
+
+In addition to previewing content within the CMS preview panel, it is also possible to preview the site from the 'front-end' (ie. as a visitor would see it).  Clicking on the URL segment in the CMS will open a new window containing the page being edited.  Code is added to the end of the URL (`stage=Stage`) which tells SilverStripe to show draft content.  
+
+This special mode is only available to logged-in users with the relevant editing permissions and will persist for the current session, or the mode is explicitly switched back by adding `stage=Live` to the end of the URL.
```

Comment:
This is actually no longer true as we've moved to having to have the param in the URL explicitly...

I admit that this is confusing as this will apply to some older installs, but our docs shouldn't be correct only for older versions :/

Reply:
I was testing on 4.1.1 and unless I've misunderstood what you mean, that behaviour isn't the way it's working. 

Once I have visited the site with ?stage=Stage I can still see the draft content even without the param.   I've got screenshots, etc. but I suspect this isn't the place for them.

Reply:
So, clarification.  This applies to the 4.2 release which is imminent, so I'll remove the relevant bits of text!